fundingmanager_test.go

```diff
@@ -2429,3 +2429,63 @@ func TestFundingManagerMaxPendingChannels(t *testing.T) {
 		t, bob.msgChan, "AcceptChannel",
 	).(*lnwire.AcceptChannel)
 }
+
+// TestFundingManagerRejectPush checks behaviour of 'rejectpush'
+// option, namely that non-zero incoming push amounts are disabled.
+func TestFundingManagerRejectPush(t *testing.T) {
+	// Enable 'rejectpush' option and initialize funding managers.
+	alice, bob := setupFundingManagers(t, defaultMaxPendingChannels)
+	rejectPush := cfg.RejectPush
+	cfg.RejectPush = true
+	defer func() {
+		tearDownFundingManagers(t, alice, bob)
+		cfg.RejectPush = rejectPush
```

Comment:
just leaving a note that this could be lead to a race condition in the unit tests if other tests are reading from `cfg` in parallel. we can revisit if it surfaces

Reply:
Hm yeah, value should probably be copied to the `fundingMgr` config.